src/main/java/io/stargate/sgv2/jsonapi/api/model/command/deserializers/SortClauseDeserializer.java

```diff
@@ -76,6 +78,9 @@ public SortClause deserialize(JsonParser parser, DeserializationContext ctxt) th
     while (fieldIter.hasNext()) {
       Map.Entry<String, JsonNode> inner = fieldIter.next();
       final String path = inner.getKey().trim();
+      // Validation will check against invalid paths, as well as decode "amp-escaping"
+      final String validatedPath = validateSortClausePath(path);
```

Comment:
This is the main fix: validation getting called for all entries (but there's change in called method too).